packages/accounts-passwordless/passwordless_server.js

```diff
@@ -80,38 +80,54 @@ Accounts.registerLoginHandler('passwordless', options => {
     Accounts._handleError('User has no token set');
   }
 
-  // This method is added by the package accounts-2fa
-  if (
-    Accounts._is2faEnabledForUser &&
-    Accounts._is2faEnabledForUser(user)
-  ) {
-    if (!options.code) {
-      Accounts._handleError('2FA code must be informed.');
-    }
-    if (
-      !Accounts._isTokenValid(user.services.twoFactorAuthentication.secret, options.code)
-    ) {
-      Accounts._handleError('Invalid 2FA code.');
-    }
-  }
-
   const result = checkToken({
     user,
     selector,
     sequence,
   });
   const { verifiedEmail, error } = result;
 
-  if (!error && verifiedEmail) {
```

Comment:
I didn't understand this removal...
This code was verifying the email if the user provided a valid passwordless token.

Reply:
I didn't realize the `verifiedEmail` was part of the verification. I've changed that.